avocado/core/nrunner/runnable.py

```diff
@@ -309,8 +309,9 @@ def _module_exists(module_name):
         mod_path = os.path.join('plugins', 'runners', module_filename)
         return pkg_resources.resource_exists('avocado', mod_path)
 
-    def pick_runner_command(self, runners_registry=None):
-        """Selects a runner command based on the runner.
+    @staticmethod
+    def pick_runner_command(kind, runners_registry=None):
```

Comment:
I might be missing something here, but I found useful to see the runner command based on a Runnable.

`runnable.pick_runner_command()` it will return the exact command to execute based on a specific runnable.

Now, with your proposal I loose this, and I have to execute this:

```python
command = Runnable.pick_runner_command(runnable.kind)
```
or 
```python
command runnable.pick_runner_command(runnable.kind) # what is redundant
```

What about keeping the method still available for the object, something like this:

```python
def runner_command(self, runners_registry=None):
   return Runnable.pick_runner_command(self.kind, runners_registry)
```

What you think?